### ISSUE TITLE
Does not work with external server

### ISSUE THREAD

**Issue by farhan3414** [InfinityWorldHI/Biometric-attendance-system-V1.0#5]
It works pefectly with the local server(xammp) but unfortunately stops working when you put yout domain name in the sketch.

**Comment by InfinityWorldHI** [comment]
Make sure the domain allow HTTP requests, Does the manage user page work on the public server?

**Comment by farhan3414** [comment]
Yes, both POST and GET HTTP requests are enabled and the manage users page also working on public server.

**Comment by neamatullah-JRC** [comment]
@farhan3414 @InfinityWorldHI is any solution here
